### PR TITLE
Portfolio-project

### PR DESCRIPTION
**Pull request by micheaol**
In this pull request I added the following:
1. Add a contact form to your portfolio with my coding partner.
2. Added name field that accepts up to 30 chars.
3. Email field accepts only emails.
4. Message field accepts up to 500 chars.
5. All fields have placeholder text.
6. All fields have the correct type.
7. A user cannot submit the form if one or more of the fields is empty.
8. Data collection by using the Formspree service.
9. use Flexbox
10. Implement the button interactions (enable, hover, etc.).

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -215,10 +215,10 @@ <h1 class="skills-header"><span><img src="./images/Rectangle-three.svg" alt=""><
         </div>
         <div class="form-container">
             <form action="https://formspree.io/f/xnqlwgep" method="POST">
-                <input type="text" name="name" placeholder="Full name">
-                <input type="email" name="email" placeholder="Email">
-                <textarea name="message" id="text-msg" cols="500" rows="5" placeholder="Enter text here"></textarea>
-                <input type="button" value="Get in touch" id="btn-submit">
+                <input type="text" name="name" placeholder="Full name" required>
+                <input type="email" name="email" placeholder="Email" required>
+                <textarea name="message" id="text-msg" cols="500" rows="5" placeholder="Enter text here" required></textarea>
```
Comment (ZeenatLawal) [inline index.html:220]: - [ ] Name input should have a maxlength="30" attribute as  stated in the requirements.
- [ ] Message input should have a maxlength="500" attribute as stated in the requirements.

File: index.html [hunk 2]
```diff
@@ -215,10 +215,10 @@ <h1 class="skills-header"><span><img src="./images/Rectangle-three.svg" alt=""><
...
+                <input type="submit" value="Get in touch" id="btn-submit">
             </form>
         </div>
```
Comment (ZeenatLawal) [inline index.html:223]: - [ ] The background colour for this section should be the same as the one in the design.

Comment (ZeenatLawal) [reply index.html:223]: - [ ] The hover state colour for the Get in touch button should be the same as the one in the design.

**Review by ZeenatLawal — CHANGES_REQUESTED** [review]
Hi @micheaol @Ntare22 and @Lino09, 

Great job so far!:clap: :clap:
## To Highlight 🎊 🎊 
- GitHub flow is followed. ✔️
- PR has a title and summary. ✔️

There are some issues that you still need to work on to go to the next project but you are almost there!

## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by emmanuellekamwa — APPROVED** [review]
**STATUS: APPROVED ✔🎉**
Hello @micheaol 👋🏽
Your project is complete!  There is nothing else to say. It's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**